# Musk-mod-0
Ejercicio Musk Modulo 7.


He corregido en Espacio_de_trabajo\EJERCICIOS\MODULO 5\T4\8.py la ruta de archivo, siendo antes una ruta fija, la he cambiado por una ruta relativa. asi no da error al ejecutarla otra persona desde el repositorio. 